eo-runtime/pom.xml

```diff
@@ -144,7 +148,8 @@ SOFTWARE.
               <targetDir>${project.basedir}/target/eo-test</targetDir>
               <addSourcesRoot>false</addSourcesRoot>
               <addTestSourcesRoot>true</addTestSourcesRoot>
-              <generatedDir>${project.basedir}/target/generated-test-sources</generatedDir>
+              <generatedDir>${project.basedir}/target/generated-test-sources
+              </generatedDir>
```

Comment:
@volodya-lombrozo it's a typo?

Reply:
@yegor256 It was `xcop --fix`. I'll return it back

Reply:
@yegor256 fixed